src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserSafeHandles.cs

```diff
@@ -18,7 +18,8 @@ internal sealed partial class SNILoadHandle : SafeHandle
         internal readonly SNINativeMethodWrapper.SqlAsyncCallbackDelegate WriteAsyncCallbackDispatcher = new SNINativeMethodWrapper.SqlAsyncCallbackDelegate(WriteDispatcher);
 
         private readonly uint _sniStatus = TdsEnums.SNI_UNINITIALIZED;
-        private readonly EncryptionOptions _encryptionOption;
+        private readonly EncryptionOptions _encryptionOption = EncryptionOptions.OFF;
+        private bool? _encryptClientPossible = null;
```

Comment:
Can we rename this to "_clientOSEncryptionSupport" instead?
Similarly "ClientOSEncryptionSupport" instead of "EncryptClientPossible"